alf/examples/ppo_pr2.sh

```diff
@@ -0,0 +1,4 @@
+CUDA_VISIBLE_DEVICES=1 python train_ppo_pr2.py --root_dir=$1 --env_name=SocialBot-Pr2Gripper-v0 --gin_param='train_eval.learning_rate=2e-4' --gin_param='PPOAgent.use_gae=True' --gin_param='PPOAgent.use_td_lambda_return=True' --num_parallel_environments=60 --num_environment_steps=30000000 --collect_episodes_per_iteration=60 --gin_param='train_eval.entropy_regularization=0.0' --gin_param='train_eval.num_epochs=10' --gin_param='train_eval.use_tf_functions=True' --gin_param='PPOAgent.importance_ratio_clipping=0.2' --gin_param='PPOAgent.check_numerics=True' --gin_param='PPOAgent.initial_adaptive_kl_beta=0' --gin_param='PPOAgent.normalize_rewards=False' --gin_param='PPOAgent.kl_cutoff_factor=0'
```

Comment:
Have you tried different proportion of collect_episodes_per_iteration and  num_epochs? Here it is 6:1 (60:10).  The original parameter in tf-aget PPO is about 1:1 (30:25). Seems to be a trade-off between sample efficiency and training time.

Reply:
collect_episodes_per_iteration should be multiple (usually 1) of parallel_environments, which I find to be one of the most critical parameters in PPO, usually the higher the better within your system limits. The original PPO paper and GAE paper by Schulman use 10 epochs. 

Another reason is: 60 episodes * 100 steps = 6k examples, I worry 25 epochs could overfit it. The current training time is still dominated by experience collection.